spk/salt-minion/src/service-setup.sh

```diff
@@ -1,33 +1,37 @@
-PYTHON_DIR="/var/packages/python3/target/bin"
+PYTHON_DIR="/var/packages/python38/target/bin"
 PATH="${SYNOPKG_PKGDEST}/bin:${SYNOPKG_PKGDEST}/env/bin:${PYTHON_DIR}:${PATH}"
 VIRTUALENV="${PYTHON_DIR}/python3 -m venv"
-PYTHON="${SYNOPKG_PKGDEST=}/env/bin/python"
+PYTHON="${PYTHON_DIR}/python3"
 LANGUAGE="env LANG=en_US.UTF-8"
 SALT_MINION="${SYNOPKG_PKGDEST}/env/bin/salt-minion"
-PID_FILE="${SYNOPKG_PKGDEST}/var/run/salt-minion.pid"
+PID_FILE="${SYNOPKG_PKGDEST}/env/var/run/salt-minion.pid"
 
-SERVICE_COMMAND="${SALT_MINION} -c ${SYNOPKG_PKGDEST}/var -d"
+SERVICE_COMMAND="${SALT_MINION} -c ${SYNOPKG_PKGDEST}/env/etc/salt -d"
 
 service_postinst ()
 {
     # Create a Python virtualenv
     ${VIRTUALENV} --system-site-packages ${SYNOPKG_PKGDEST}/env
 
-    # Install wheels
+    # Install the wheels
     wheelhouse=${SYNOPKG_PKGDEST}/share/wheelhouse
-    ${SYNOPKG_PKGDEST}/env/bin/pip install --no-deps --force-reinstall --no-index --find-links ${wheelhouse} ${wheelhouse}/*.whl
+    ${SYNOPKG_PKGDEST}/env/bin/pip install --no-deps --no-index --upgrade --force-reinstall -f ${wheelhouse} ${wheelhouse}/*.whl
```

Comment:
again, new functions to do that:
https://github.com/SynoCommunity/spksrc/wiki/Using-wheels-to-distribute-Python-packages

Be sure to have your PATH variables set accordingly as well.